src/main.css

```diff
@@ -222,3 +222,12 @@ body,
 	-webkit-user-select: none;
 	user-select: none;
 }
+
+/* 
+ Used for pdf printing. Prints only elements inside #html-print
```

Comment:
`Used for PDF printing directly from a component, e.g., FeatureInfoPanel. Prints only elements inside #html-print`